### ISSUE TITLE
Information messaging in library signup/login

### ISSUE THREAD

**Issue by s-steddy** [Scholar-6/brillder#4689]
xyz

**Comment by ivanromakh** [comment]
I will add login button for new library login.
![image](https://user-images.githubusercontent.com/18567511/188597633-38fb7837-3a10-475d-8dd5-247917292517.png)

**Comment by ivanromakh** [comment]
sign up
![image](https://user-images.githubusercontent.com/18567511/188609229-90c2dca5-76da-4696-bdce-0b6c7b9aafdc.png)
login
![image](https://user-images.githubusercontent.com/18567511/188609974-f8cf30b5-77ab-4f61-aa68-9118bf4aab23.png)

**Comment by ivanromakh** [comment]
Register updated like in the image above.
![image](https://user-images.githubusercontent.com/18567511/188841569-6744cf0a-6e8f-49c5-9abe-b5ae14963827.png)

**Comment by s-steddy** [comment]
Some text explaining why we need email and password from new users. What do you think the best way to present this info will be?

Thanks for signing up to Brillder! Please register an email and password to your account.

Brillder does not send general publicity messages, and we will only contact you in specific circumstances, for example in case of an issue with your account. If you need to contact us, this also helps us verify who you are. Your email address will be securely stored, and will never be shared with a third party. Our full terms and conditions will be shown shortly, or you can click [here](https://dev-app.brillder.com/terms).

**Comment by ivanromakh** [comment]
not sure about
Thanks for signing up to Brillder!
as he didn`t signed up yet. he need to fill email to sign up and password.

**Comment by s-steddy** [comment]
Good point. But let's make it clear they only have to give their email once as part of onboarding for new users. How about this, or is it too long?

Thank you, your credentials have been verified. As a new user, please register an email and password to your account.

**Comment by ivanromakh** [comment]
nice

**Comment by ivanromakh** [comment]
@s-steddy 
![image](https://user-images.githubusercontent.com/18567511/188911929-b7525861-12f3-40bb-b117-326e68ac3cb5.png)

**Comment by s-steddy** [comment]
Going to leave some comments here and in #4688 so there's something written down. 

A few fixes remain for the page you arrive at from this link, but I'll record them in #4688, otherwise from this page we need to make sure the separate signup/login flows are working properly:
 http://dev-app.brillder.com/play/brick/1714/cover/history-politics/greenham-common-womens-peace-camp?referralId=cyxlstautp&origin=library

FOR SIGNUP PAGE
Barcode/pin entry page is good, but after that you get to this page:
https://files.slack.com/files-pri/TT0LJEP8X-F042A5ED76C/screenshot_2022-09-08_at_16.01.03.png
- [ ] Welcome popup should read: 
     **Thank you, your credentials have been verified** 
     As a new user, please register an email and password to your account.
- [ ] Remove equivalent text from above email/password fields.
- [ ] At bottom of this popup, green button that says OK that clears the popup. You can also just click out of the popup.

If popup can have expading text inside, ie. this function, then put the explanation of why email/password are needed in the popup.
<img width="199" alt="Screenshot 2022-09-08 at 17 29 32" src="https://user-images.githubusercontent.com/107257153/189176200-95ecf7f3-473d-4643-8263-cb1e5985c489.png">
- [ ] Text: How will this information be used? >
- [ ] On expand: Registering an email address ensures you can still track your achievements should your library membership change, for example were you to move from your current area. Brillder does not send general publicity messages, and we will only contact you in specific circumstances, such as in case of an issue with your account. If you need to contact us, this also helps us verify who you are. Your email address will be securely stored, and will never be shared with a third party. Our full terms and conditions will be shown shortly, or else see [Terms](https://dev-app.brillder.com/terms) at the foot of this page.

- [ ] Otherwise, keep "How will this information be used?" as text underneath the Sign Up button, but we should try to get the expanded text to not cover the email/password fields.

- [ ] With unsused barcode/pin - Good, continue to onboarding
- [ ] With already used barcode/pin, error popup with this message:
     **Authentication Unsuccessful**
     These credentials have already been connected to an account. Did you mean to [Sign In] (link to sign in page)? Please contact us if this doesn't seem right.

FOR LOGIN PAGE
On barcode/pin entry page:
- [ ] Correct and registered barcode and pin - good, redirect to **Brick** (remember not account home or catalogue)
- [ ] Incorrect registered barcode/pin or unrecognised barcode/pin (guess we can't distinguish) - error popup with message:
     **Authentication Unsuccessful**
     Please check the information you entered was correct, unless you intended to [Sign Up] (link to register page)? Otherwise, please contact us if this doesn't seem right.

**Comment by ivanromakh** [comment]
text
![image](https://user-images.githubusercontent.com/18567511/189329760-11ece46e-36b2-4d8f-a134-dae827952c8a.png)
